operator/deploy/role.yaml

```diff
@@ -72,6 +73,8 @@ rules:
   resources:
   ${OPERATOR_ROLE_CLUSTERROLEBINDINGS}
   ${OPERATOR_ROLE_CLUSTERROLES}
+  - clusterroles
```

Comment:
This is what we need to review, that cache should work well in clusterroles or no elevated privileges scenarios.
So, that may affect some queries (namespaces/projects), but other objects can get benefit of them.